### ISSUE TITLE
No graceful way to exit the home share page

### ISSUE THREAD

**Issue by troywa** [ventuso/Fi-Test#99]
IPhone 5c version 8.2
Fi 1.0.42

Don't see a  graceful way to exit back to the map or list pages once you click on Home Shares.
Steps to reproduce:
I'm connected to Range network
Click on the hamburger icon.
Click on Home Shares
Don't see  real easy way to exit.
From the Home Shares window I can click on the hamburger icon again.
Then I can click on the green "Go To Network" button.  This will take me back to the main Range list page but the Map/List options at the top of the page is gone.
I have to click on Build Network.  Then click on the x where it says "Want Fi Here?" to get back to the Map/List map page.

![img_0168](https://cloud.githubusercontent.com/assets/11478454/7244287/dd56f820-e79e-11e4-9f7a-6d12dde26d54.PNG)

**Comment by baonguyenventuso** [comment]
I agree on this, We need a way to get back to the main Map (Discover) from the HomeShare, we are currently can do that on the Menu only and I still think this flow is not a good way to go:

1. Finish share on HomeSharing 
2. Go to Menu 
3. Tap on Build 
4. Tap on Discover 

I think we need a button on HomeSharing to come back to Discover. Beside, I still think HomeSharing need a specific design for it. Check out on Android, it doesn't look good. Please help us on that. 

Best regards,

**Comment by baonguyenventuso** [comment]
We will implement this in Release 2 (Hold it here for release 1)
